packages/@lwc/shared/src/__tests__/html-attributes.spec.ts

```diff
@@ -4,34 +4,93 @@
  * SPDX-License-Identifier: MIT
  * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
  */
-import { htmlPropertyToAttribute } from '../html-attributes';
+import { htmlAttributeToProperty, htmlPropertyToAttribute } from '../html-attributes';
 
-describe('htmlPropertyToAttribute', () => {
-    test.each([
-        // Standard attribute mapping
-        ['foo', 'foo'],
-        ['fooBar', 'foo-bar'],
-        ['fooBarBaz', 'foo-bar-baz'],
-        ['FooBar', '-foo-bar'],
+const propToAttr = [
+    // Standard attribute mapping
+    ['foo', 'foo'],
+    ['fooBar', 'foo-bar'],
+    ['fooBarBaz', 'foo-bar-baz'],
+    ['FooBar', '-foo-bar'],
+
+    // Aria attribute mapping
+    ['ariaActiveDescendant', 'aria-activedescendant'],
+    ['ariaAtomic', 'aria-atomic'],
+    ['ariaAutoComplete', 'aria-autocomplete'],
+    ['ariaBusy', 'aria-busy'],
+    ['ariaChecked', 'aria-checked'],
+    ['ariaColCount', 'aria-colcount'],
+    ['ariaColIndex', 'aria-colindex'],
+    ['ariaColSpan', 'aria-colspan'],
+    ['ariaControls', 'aria-controls'],
```

Comment:
Just FYI, some of these are an error due to making a bet on a draft spec. See: https://github.com/salesforce/lwc/issues/2733

Assuming we ever fix that, some of these tests will have to change.